flask_app.py

```diff
@@ -1,3 +1,30 @@
 """
 Put your Flask app code here.
-"""
\ No newline at end of file
+"""
+
+from flask import Flask, request, render_template
+app = Flask(__name__)
+#Make an app.route() decorator here
+@app.route("/initial/", methods = ['GET', 'POST'])
```

Comment:
It might be better to split get and post into different routes, however this should be fine as well